BitTornado/Meta/Info.py

```diff
@@ -230,6 +230,8 @@ def __init__(self, name, size=None,
             self['files'] = []
             self['length'] = size
 
+        self['private'] = params['private'] if 'private' in params else False
```

Comment:
Unfortunately making it optional didn’t work, again because of the strict rules in the constrained directories. The following code:

    from BitTornado.Meta.Info import Info, MetaInfo
    from BitTornado.Meta.BTTree import BTTree
    from BitTornado.Meta.bencode import bencode

    tree = BTTree('test.txt', ['test.txt'])
    info = Info('test_torrent', tree.size)
    tree.addFileToInfos((info,))
    metainfo = MetaInfo(announce='http://tracker/', info=info)
    bencode(metainfo)

Fails with this error:

    Traceback (most recent call last):
      File "test.py", line 11, in <module>
        bencode(metainfo)
      File "/home/sam/bittornado/BitTornado/Meta/bencode.py", line 36, in __call__
        self.encode(data, ctext)
      File "/home/sam/bittornado/BitTornado/Meta/bencode.py", line 60, in encode
        self.encode(data, ctext)
      File "/home/sam/bittornado/BitTornado/Meta/bencode.py", line 56, in encode
        for key, data in sorted(ilist):
      File "/home/sam/bittornado/BitTornado/Meta/Info.py", line 315, in <genexpr>
        return ((key, self[key]) for key in self.keys())
      File "/home/sam/bittornado/BitTornado/Meta/Info.py", line 298, in __getitem__
        return super(Info, self).__getitem__(key)
    KeyError: 'private'